_pkgdown.yml

```diff
@@ -131,3 +138,10 @@ reference:
     - theme_bootswatch
     - bootswatch_themes
     - version_default
+
+
+  redirects:
+    - ["articles/cards.html#fixed-sizing", "articles/cards.html#scrolling-contents"]
```

Comment:
Unfortunately I don't think pkgdown's redirection system works for within-page redirects. IIRC it's a `<meta http-equiv="refresh">` kind of redirect, so it can only redirect from one page to another.